modules/csg/csg.h

```diff
@@ -135,6 +135,17 @@ struct CSGBrushOperation {
 				return h;
 			}
 		};
+		struct Intersection {
+			bool found = false;
+			real_t conormal = 9999999.0f; // Should be MAX, but probably is never larger than 1.
+			real_t distance_squared = 9999999.0f; // Should be MAX.
+			real_t origin_angle = 9999999.0f; // Should be MAX, but probably is never larger than 1.
```

Comment:
Would it make sense to use `FLOAT_MAX`, or is it overkill?

Reply:
I wondered too. This number is not a constant, but I am in the middle of something. The test plan is to check your sample and the zip for the change to float max and the precision of snapping.

Reply:
I changed it to `FLT_MAX`, seems to work.